docs/sphinx/source/whatsnew/v0.10.2.rst

```diff
@@ -62,6 +62,8 @@ Documentation
   (:issue:`1724`, :pull:`1838`)
 * Update definition of snow events parameter for :py:func:`pvlib.snow.loss_townsend`.
   (:issue:`1839`, :pull:`1840`)
+* Typo fix in :py:func:`pvlib.temperature.fuentes`, `wind_height` parameter description.
+  (:issue:`1814`, :pull:`1860`)
```

Comment:
Given that the change has no impact, I would leave it out of the whatsnew to avoid cluttering.

This is probably a personal standpoint, but let's see if others agree.

Reply:
I always doubt. Just did it because of past experience with #1833, although it's true that this is not as important as an example.

Feel free to revert or delete that entry.

Reply:
For the change description, I think #1833 was borderline about whether someone might care, and I think this one can be safely omitted.  For things like this the main thing is to make sure to list all contributors (which you have already done 👍 ).